bindata/etcd/etcd-restore-backup.sh

```diff
@@ -11,77 +12,50 @@ if [[ $EUID -ne 0 ]]; then
   exit 1
 fi
 
-usage () {
-    echo 'Path to the directory containing backup files is required: ./etcd-snapshot-restore.sh <path-to-backup>'
-    exit 1
+function usage {
+  echo 'Path to the directory containing backup files is required: ./etcd-snapshot-restore.sh <path-to-backup>'
```

Comment:
this isn't descriptive enough for me. What content is expected here

Reply:
Well, our current backup is expected to be in a directory containing two components:
1. etcd snapshot
1. Bundle of Static POD resources (which includes kube-apiserver's static pod resources, that may include encryption key)

Reply:
> Well, our current backup is expected to be in a directory containing two components:
> 
> 1. etcd snapshot
> 2. Bundle of Static POD resources (which includes kube-apiserver's static pod resources, that may include encryption key)

don't tell me, write in the usage for everyone :)

Reply:
Added to the usage.